src/Grid.cpp

```diff
@@ -0,0 +1,322 @@
+//
+//  Grid.cpp
+//  Game of Life
+//
+//  Created by Alex Frasca on 01/05/2023.
+//
+
+#include "Grid.hpp"
+
+namespace conway {
+
+// Cell
+
+Cell::Cell(sf::Vector2<int> idx, sf::Vector2<float> pos, float spc, CELL_STATE st) {
+    index = idx;
+    position = pos;
+    spacing = spc;
+    state = st;
+    kill_next_cycle = false;
+    activate_next_cycle = false;
```

Comment:
Use member list initialization for data members being initialized through arguments, and initialize the rest in the class definition itself.
```cpp
// member list init:
Cell::Cell(sf::Vector2<int> idx, sf::Vector2<float> pos, float spc, CELL_STATE st) 
  : index(idx), positon(pos), spacing(spc), state(st) {}

// in class definition (this is already happening, just demonstrating how 
// you'd do it for true / any other values):
    bool kill_next_cycle{false};
    bool activate_next_cycle{false};
```